### DISCUSSION TITLE
Reserve for The Team

### DISCUSSION THREAD

**Discussion by steigenn** [hashlips-lab/nft-erc721-collection#432] [Q&A]
Hello, I would be very grateful if you could help me with my problem. 

https://www.youtube.com/watch?v=edYfMHnsI4I

1) I want to reserve some nft for team. It is mentioned in the video that it is difficult to allocate a large number of reserves.. What is the estimated maximum amount?

2) Do I reserve the reserve I want for my own wallet with the code shown in the video? Also, am I paying high gas fees when I do this?

3) What happens when everything goes well? Are nfts minted in my wallet for free? If this is not true, how is the process progressing? and if these are false how do I mint the reserved reserve?
_________________________________________________

4) Should I enter my wallet address in the address section in the photo?

![ss](https://user-images.githubusercontent.com/119765600/208441021-12b0b794-7d6d-436f-8df2-617056329010.PNG)

@liarco

**Comment by web3chick** [comment]
> Hello, I would be very grateful if you could help me with my problem.
> 
> https://www.youtube.com/watch?v=edYfMHnsI4I
> 
> 1. I want to reserve some nft for team. It is mentioned in the video that it is difficult to allocate a large number of reserves.. What is the estimated maximum amount?
> 2. Do I reserve the reserve I want for my own wallet with the code shown in the video? Also, am I paying high gas fees when I do this?
> 3. What happens when everything goes well? Are nfts minted in my wallet for free? If this is not true, how is the process progressing? and if these are false how do I mint the reserved reserve?
> 
> 4. Should I enter my wallet address in the address section in the photo?
> 
> ![ss](https://user-images.githubusercontent.com/119765600/208441021-12b0b794-7d6d-436f-8df2-617056329010.PNG)
> 
> @liarco

After the deployment to Goerli, write the contract and use the mintForAddress function. You can mint to your own address or any other address that is considered the reserve wallet. Yes, it is free since that function can only be called by the owner. The maximum you can reserve depends on the maximum mint amount specified in the contract. It is also advisable not to mint too many at once so it doesn't get flagged as spam by Opensea.

**Reply by liarco** [reply]
Hi @web3chick, thx for reaching out to us.
We are super busy at the moment so our public support is limited to bug fixing and security updates.

For anyone looking for dedicated support we are giving priority to holders of the [Open Devs Crew](https://opendevs.io) token in order to sustain the development of our tools and focus our resources on contributors. We have a dedicated support channel for holders on the HashLips Discord server.

**Comment by caojunhua** [comment]
你好，我已经收到了！
